assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/text_to_image/conda.yaml

```diff
@@ -8,6 +8,7 @@ dependencies:
 - pip:
   - mlflow==2.6.0
```

Comment:
Is this latest from main? We have recently updated mlflow version.

Reply:
It was not. Merged from master, thanks.